packages/@aws-cdk/aws-eks/README.md

```diff
@@ -269,6 +269,8 @@ the capacity.
 The `KubernetesManifest` construct or `cluster.addManifest` method can be used
 to apply Kubernetes resource manifests to this cluster.
 
+> When using `cluster.addManifest`, the manifest is created inside the cluster stack. If the manifest contains attributes from a different stack that depends on the cluster stack, a circular dependency will be created and you will get a synth time error. To avoid this, directly use `new KubernetesManifest` to create the manifest in the other stack.
+
```

Comment:
```suggestion
> When using `cluster.addManifest`, the manifest construct is defined within the cluster's stack scope. If the manifest contains attributes from a different stack which depend on the cluster stack, a circular dependency will be created and you will get a synth time error. To avoid this, directly use `new KubernetesManifest` to create the manifest in the scope of the other stack.

```

Reply:
👍